src/main/java/com/conveyal/gtfs/loader/JdbcTableWriter.java

```diff
@@ -171,9 +163,8 @@ public String update(Integer id, String json, boolean autoCommit) throws SQLExce
             if (specTable.name.equals("patterns")) {
                 referencingTables.add(Table.SHAPES);
             }
-
-            boolean referencedPatternUsesFrequencies = referencedPatternUsesFrequencies(connection, jsonObject);
-
+            PatternReconciliation.initialize(connection, tablePrefix);
```

Comment:
Oh, I thought you would write something like `PatternReconciliation reconciliation = new PatternReconciliation(...)`!

Reply:
er yeah, that does make more sense! Have addressed.